Cargo.toml

```diff
@@ -50,6 +50,8 @@ anstream = "0.6.13"
 assert_cmd = "2.0"
 similar-asserts = { version = "1.5.0", features = ["serde"] }
 predicates = "3.1.0"
+insta = { version = "1.39.0", features = ["yaml"] }
```

Comment:
Personally, I somewhat prefer `ron` over `yaml` since it's more explicit and ergonomic around Rust (ADT-style) enums. What do you think?

Reply:
That's fine with me, I was going off of the [docs](https://insta.rs/docs/serializers/):

> This serializer is recommended because YAML is human readable and excellent at diffing because it is line based.

I think ron would work (is it more source control friendly?) as well.

Reply:
Ultimately, let's pick whatever looks the best for the job. I love that you opened a PR with a specific test case where we can look at the exact generated output and decide based on that.